internal/dcache/replication_manager/replication_manager.go

```diff
@@ -517,13 +675,20 @@ retry:
 			// chunks which we could not write to this component RV will be later sync'ed
 			// from one of the good component RVs.
 			//
-			log.Warn("ReplicationManager::WriteMV: Writing to %s/%s on node %s failed, marked RV offline",
-				respItem.rvName, req.MvName, respItem.targetNodeID)
+			log.Warn("ReplicationManager::WriteMV: Writing to %s/%s failed, marked RV offline",
+				respItem.rvName, req.MvName)
 			continue
 		}
 
+		//
 		// The error is RPC error of type *rpc.ResponseError.
-		if rpcErr.GetCode() == models.ErrorCode_NeedToRefreshClusterMap {
+		//
+		if rpcErr.GetCode() == models.ErrorCode_BrokenChain {
+			brokenChain = true
```

Comment:
see if we can easily assert that this error won't happen for nexthop

Reply:
This can happen in the next try if there are nodes going down in incremental order. For example, in the first try rv1 went down, so we marked that as offline and skipped the PutChunk for next RVs. In the next retry, suppose rv2 went offline. So, we can get broken chain error for RVs after rv2 and this can go on. This is why I am not incrementing the retry count in case of broken chain error.

Reply:
What I mean is - we should never get this error from our nexthop to which we sent the PutChunkDC request